generators/liquibase/templates/gradle/liquibase.gradle.ejs

```diff
@@ -5,6 +5,7 @@ configurations {
 dependencies {
   implementation "org.liquibase:liquibase-core"
   liquibaseRuntime "org.liquibase:liquibase-core"
+  liquibaseRuntime "info.picocli:picocli:4.6.1"
```

Comment:
```suggestion
  // Dependency required to parse options. Refer to https://github.com/liquibase/liquibase-gradle-plugin/tree/Release_2.2.0#news.
  liquibaseRuntime "info.picocli:picocli:4.6.1"
```

Reply:
@hide212131 Can you add picocli to the [pom](https://github.com/jhipster/generator-jhipster/blob/main/generators/server/resources/pom.xml) and manage the version update do be done automatically?

Reply:
@DanielFran  I didn't know about the additional comments and just now pushed it. I will check it out.